### PR TITLE
Fixed the shift clicking to select any layer that is clicked on

### PR DESCRIPTION
**Pull request by DivyaPrabhakar**
Changed the event policy for SHIFT to send events to Ds. Regarding issue #2069

### CODE REVIEW

File: src/js/actions/tools.js [hunk 1]
```diff
@@ -175,7 +175,7 @@ define(function (require, exports) {
             outsideShiftPolicy = new PointerEventPolicy(adapterUI.policyAction.ALWAYS_PROPAGATE,
                 adapterOS.eventKind.LEFT_MOUSE_DOWN,
                 {
-                    shift: true
+                    shift: false
```
Comment (volfied) [inline src/js/actions/tools.js]: So modifiers table is not a true/false table. It's more of an existence thing. 

In this bug's case, you don't want to disable shift, you want shift clicks to all come to us, which you do by changing the propagation action of this policy (to NEVER)

**Comment by volfied** [comment]
@DivyaPrabhakar so close!

**Comment by DivyaPrabhakar** [comment]
@volfied Got it, hope this is the correct change now.

**Comment by volfied** [comment]
:signal_strength:

**Pull request merged**